tests/test_tasks.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the user_tasks subclasses of celery.Task."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from celery import shared_task, Task
+import pytest
+
+from user_tasks.models import UserTaskStatus
+from user_tasks.tasks import UserTask, UserTaskMixin
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
Having the logger already configured makes it really easy to start logging info when debugging (especially if I need to log from a few different files), so I tend to leave it in any file I've ever needed to add it to.  Do you think it's preferable to rip it out between debugging sessions?